test/config/karma.unit.js

```diff
@@ -90,10 +92,15 @@ module.exports = function(karma) {
   };
 
   if (collectTranslations) {
-    config.plugins = [].concat(config.plugins || [], 'karma-*', require('./translation-reporter'));
+    config.plugins = [].concat(config.plugins || ['karma-*'], require('./translation-reporter'));
     config.reporters = [].concat(config.reporters || [], 'translation-reporter');
     config.envPreprocessor = [].concat(config.envPreprocessor || [], 'COLLECT_TRANSLATIONS');
   }
 
+  if (singleStart) {
+    config.browsers = [].concat(config.browsers, [ 'Debug' ]);
```

Comment:
(minor) sometimes in this file you use `.concat([ 'something' ]);` in different places we use `.concat('something');`.
Obviosuly yielding the same results, yet while we touch it, we might refactor it to be consistent. (Would help me at least when reading the code)